.github/workflows/integration.yml

```diff
@@ -0,0 +1,36 @@
+name: Integration Test
```

Comment:
Instead of adding a new test suite, why not just use the existing Test workflow that just executes `integration-test` instead of `test`? AFAIK, the normal unit tests also run, and so we get only 1 coverage report.